packages/datadog-plugin-mocha/src/index.js

```diff
@@ -122,6 +122,7 @@ function getAllTestsInSuite (root) {
 function createWrapRunTests (tracer, testEnvironmentMetadata, sourceRoot) {
   return function wrapRunTests (runTests) {
     return function runTestsWithTrace () {
+      this.once('end', () => tracer._exporter._writer.flush())
```

Comment:
Any way that this can be tested?

Reply:
not that I can think of, no. The issue is intermittent and only happens when running mocha in parallel mode. `flush` is called normally so I don't know if we could just assert on it being called. This has been tested manually by the customer and me and it seems to work.